packages/core/src/Stores/client-store.js

```diff
@@ -462,6 +462,16 @@ export default class ClientStore extends BaseStore {
         return this.residence === 'gb';
     }
 
+    @computed
+    get is_im() {
```

Comment:
normally we use is_eu or is_uk, since we have regulators specifically for that, for these cases, its better to check residence straight in the isolated usage 🙏

Reply:
Ok, I removed these functions from here